sdcm/cluster.py

```diff
@@ -347,25 +351,23 @@ def destroy(self):
         pass
 
 
-class BaseNode():  # pylint: disable=too-many-instance-attributes,too-many-public-methods
+class BaseNode(AutoSshContainerMixin, WebDriverContainerMixin):  # pylint: disable=too-many-instance-attributes,too-many-public-methods
```

Comment:
why `WebDriverContainerMixin` should be part of BaseNode. Base Node is base for all?